src/components/Tables/Table.tsx

```diff
@@ -135,28 +258,164 @@ const Table: React.FC<Props> = (props) => {
-            subHeader
-            subHeaderComponent={
-                <TableFilterBar
-                    filterText={filterText}
-                    onFilter={onFilter}
-                    handleClear={handleClear}
-                    headers={props.headers}
+        <Styling>
+            <NoSsr>
+                <DataTable
+                    // types are fine without the cast when not using styled components, not sure what's happening here
+                    columns={columns}
+                    data={dataToFilteredRows(data, filterText, props.headers)}
+                    keyField="rowId"
+                    fixedHeader
+                    subHeader
+                    subHeaderComponent={
+                        <TableFilterBar
+                            filterText={filterText}
+                            filterKeys={filterKeys}
+                            toggleableHeaders={props.toggleableHeaders}
+                            onFilter={onFilter}
+                            handleClear={handleClear}
+                            headers={props.headers}
+                            setShownHeaderKeys={setShownHeaderKeys}
+                            shownHeaderKeys={shownHeaderKeys}
+                        />
+                    }
+                    pagination={props.pagination ?? true}
+                    persistTableHead
                 />
-            }
-            pagination
-            persistTableHead
-        />
+            </NoSsr>
+        </Styling>
     );
 };
 
+const EditandReorderArrowDiv = styled.div`
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    width: 100%;
+    // this transform is necessary to make the buttons visually consistent with the rest of the table without redesigning the layout
+    transform: translateX(-0.8rem);
+`;
+
+const StyledIcon = styled(FontAwesomeIcon)`
+    cursor: pointer;
+`;
+
+const Styling = styled.div`
+    // the component with the filter bars
+    & > header {
+        background-color: transparent;
+        display: flex;
+        align-items: center;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        padding: 10px;
+        gap: min(1rem, 5vw);
+        overflow: visible;
+        @media (min-width: 500px) {
+            flex-wrap: nowrap;
+        }
+
+        // the clear button
+        & > button {
+            border-radius: 0.4rem;
+        }
+    }
+
+    // the entire table component including the header and the pagination bar
+    & > div {
+        border-radius: 0;
+        background-color: transparent;
+
+        // the pagination bar
+        > nav {
+            background-color: transparent;
+            color: ${(props) => props.theme.surfaceForegroundColor};
+            border-top: solid 1px ${(props) => props.theme.foregroundColor};
+        }
+    }
+
+    // the icons in the pagination bar
+    & svg {
+        // TODO: VFB-16 different greys for different theme modes
```

Comment:
There are quite a few TODOs and comments. Does it need to all be here?

Reply:
The comments are the only thing that make the css maintainable and understandable as I'm targeting components coming from a library. The todos are for Chloe who's doing theming.